test/ectest.c

```diff
@@ -188,1185 +146,1039 @@ static void group_order_tests(EC_GROUP *group)
+            "0000000000000000008000000000000000000001",
+        "0000000000000000000000000000000000000000000000000000000000000000"
+            "0000000000000000000000000000000000000001",
+        "0021A5C2C8EE9FEB5C4B9A753B7B476B7FD6422EF1F3DD674761FA99D6AC27C8"
+            "A9A197B272822F6CD57A55AA4F50AE317B13545F",
+        "015D4860D088DDB3496B0C6064756260441CDE4AF1771D4DB01FFE5B34E59703"
+            "DC255A868A1180515603AEAB60794E54BB7996A7",
+        "0061B1CFAB6BE5F32BBFA78324ED106A7636B9C5A7BD198D0158AA4F5488D08F"
+            "38514F1FDF4B4F40D2181B3681C364BA0273C706",
+        1,
+        "010000000000000000000000000000000000000000000000000001E2AAD6A612"
+            "F33307BE5FA47C3C9E052F838164CD37D9A21173",
+        "2", 409
+    },
+    /* Curve K-571 (FIPS PUB 186-2, App. 6) */
+    {
+        "NIST curve K-571",
+        "8000000000000000000000000000000000000000000000000000000000000000"
+            "0000000000000000000000000000000000000000000000000000000000000000"
+            "000000000000425",
+        "0",
+        "1",
+        "026EB7A859923FBC82189631F8103FE4AC9CA2970012D5D46024804801841CA4"
+            "4370958493B205E647DA304DB4CEB08CBBD1BA39494776FB988B47174DCA88C7"
+            "E2945283A01C8972",
+        "0349DC807F4FBF374F4AEADE3BCA95314DD58CEC9F307A54FFC61EFC006D8A2C"
+            "9D4979C0AC44AEA74FBEBBB9F772AEDCB620B01A7BA7AF1B320430C8591984F6"
+            "01CD4C143EF1C7A3",
+        0,
+        "0200000000000000000000000000000000000000000000000000000000000000"
+            "00000000131850E1F19A63E4B391A8DB917F4138B630D84BE5D639381E91DEB4"
+            "5CFE778F637C1001",
+        "4", 571
+    },
+    /* Curve B-571 (FIPS PUB 186-2, App. 6) */
     {
-        const EC_POINT *points[4];
-        const BIGNUM *scalars[4];
-        BIGNUM *scalar3;
+        "NIST curve B-571",
+        "8000000000000000000000000000000000000000000000000000000000000000"
+            "0000000000000000000000000000000000000000000000000000000000000000"
+            "000000000000425",
+        "0000000000000000000000000000000000000000000000000000000000000000"
+            "0000000000000000000000000000000000000000000000000000000000000000"
+            "0000000000000001",
+        "02F40E7E2221F295DE297117B7F3D62F5C6A97FFCB8CEFF1CD6BA8CE4A9A18AD"
+            "84FFABBD8EFA59332BE7AD6756A66E294AFD185A78FF12AA520E4DE739BACA0C"
+            "7FFEFF7F2955727A",
+        "0303001D34B856296C16C0D40D3CD7750A93D1D2955FA80AA5F40FC8DB7B2ABD"
+            "BDE53950F4C0D293CDD711A35B67FB1499AE60038614F1394ABFA3B4C850D927"
+            "E1E7769C8EEC2D19",
+        "037BF27342DA639B6DCCFFFEB73D69D78C6C27A6009CBBCA1980F8533921E8A6"
+            "84423E43BAB08A576291AF8F461BB2A8B3531D2F0485C19B16E2F1516E23DD3C"
+            "1A4827AF1B8AC15B",
+        1,
+        "03FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
+            "FFFFFFFFE661CE18FF55987308059B186823851EC7DD9CA1161DE93D5174D66E"
+            "8382E9BB2FE84E47",
```

Comment:
Not sure I agree with this kind of indentation...  I can see the point, but...  If readability is the issue, then I'd prefer an added blank line between values.